paywall/paywall-builder/build.js

```diff
@@ -24,9 +24,16 @@ export default function buildPaywall(window, document, lockAddress, blocker) {
     return
   }
 
+  // in the paywall, postMessage needs to know the origin of the parent window
+  // This allows us to securely pass it in
+  const originUrl = `?origin=${encodeURIComponent(window.origin)}`
   const paywallUrl =
```

Comment:
note that `paywallUrl` is still used in order to redirect to the paywall in a new window. So now there is a distinction between the URL that passes in origin (only in iframe) and doesn't (only when paywall opens in separate window)